arcaflow_plugin_horreum_client/horreum_client_plugin.py

```diff
@@ -34,9 +34,19 @@ def horreum_client(
         auth_return.raise_for_status()
         token = json.loads(auth_return.text)["access_token"]
 
-    except (requests.ConnectionError, requests.HTTPError, requests.Timeout):
+    except requests.exceptions.SSLError as e:
         return "error", ErrorOutput(
-            f"Error communicating with the Keycloak server: {auth_return.text}"
+            f"Unable to establish SSL connection with the Keycloak server: {e}"
+        )
+
+    except (requests.ConnectionError, requests.Timeout) as e:
+        return "error", ErrorOutput(
+            f"Error communicating with the Keycloak server: {e}"
+        )
```

Comment:
Since each of the messages at lines 39 and 45 include the stringized exception object, unless that stringization isn't very useful, providing separate handling for the two sets of exceptions isn't worth the code complication, since they don't really do anything different from each other.  (The pertinent points are that there was an error, it was in connecting to the Keycloak server, and the specific details follow...you don't need two `except` clauses to report that.)

The analogous comment applies to lines 74 and 79.